### DISCUSSION TITLE
2022 AIGO 比賽開跑

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#519] [General]
## 比賽官網
2022 AIGO 比賽開跑了，有興趣的朋友可以去AIGO關往看看今年的比賽題目：https://aigo.org.tw/zh-tw/competitions

## 比賽時程
![image](https://user-images.githubusercontent.com/1768891/160072400-942011f4-c2e1-4586-9f75-b700af558709.png)

## 我覺得感興趣的題目
1. https://aigo.org.tw/zh-tw/competitions/details/426
2. https://aigo.org.tw/zh-tw/competitions/details/429
3. https://aigo.org.tw/zh-tw/competitions/details/430
4. https://aigo.org.tw/zh-tw/competitions/details/432
5. https://aigo.org.tw/zh-tw/competitions/details/440
6. https://aigo.org.tw/zh-tw/competitions/details/441
7. https://aigo.org.tw/zh-tw/competitions/details/442
8. https://aigo.org.tw/zh-tw/competitions/details/452
9. https://aigo.org.tw/zh-tw/competitions/details/468
10. https://aigo.org.tw/zh-tw/competitions/details/469
11. https://aigo.org.tw/zh-tw/competitions/details/482
12. https://aigo.org.tw/zh-tw/competitions/details/479

## 提問
大家有感興趣的題目嗎? 貼在下面一起討論~

**Comment by HuanYuuuuuu** [comment]
獎金有70萬元 倒是很吸引我 但可惜是4月到10月 今年可能沒有辦法參加
